CHANGELOG.md

```diff
@@ -103,7 +104,7 @@ In addition to all of the updates from Improbable, this release includes x impro
 
 ## [`0.3.0`](https://github.com/spatialos/UnrealGDK/releases/tag/0.3.0) - 2019-04-04
 
-### New Known Issues: 
+### New Known Issues:
```

Comment:
Bunch of random space deletions in this file?

Reply:
I think it is my editor config that trims the files by default